### ISSUE TITLE
Using Validation series while training RNN and Backtesting for SMAPE

### ISSUE THREAD

**Issue by SireeshLimbu** [unit8co/darts#1562]
I have a time series pandas dataframe (`Train_t`) with 20 features, which includes one target and 19 covariates/columns, each having 39 months/rows of data. DateTime starts from **'2019-08-01' - '2022/10/01'**. I am trying to divide into 24 train and 15 validation months to calculate the SMAPE of the training. But as I enter the validation series I get an error saying` 'The provided validation time series dataset is too short for obtaining even one training point.'`

```
#Trying to divide dataset into Training and Validation sets for the training.
Train_t_X = TimeSeries.from_dataframe(Train_t,value_cols=list(Train_t.columns.drop("target")),freq="1MS")
X_train_t, X_val_ = Train_t_X.split_after(pd.Timestamp("2021-07-01"))

Train_t_Y = TimeSeries.from_dataframe(Train_t,freq = '1MS',value_cols = 'target')
y_train_t, y_val_ = Train_t_Y.split_after(pd.Timestamp("2021-07-01"))

#Initializing model and fitting
model = BlockRNNModel(input_chunk_length=18,model = 'GRU',dropout=0.2,random_state = 42,output_chunk_length = 6)

#ERROR HERE
history = model.fit(y_train_t,past_covariates = X_train_t, val_series=y_val_, val_past_covariates = X_val_, verbose=False)

pred = model.predict(n=6,past_covariates = X_train_t,verbose = False)

```

Next, If I ignore the train test-split and train the model on all the 39 months (input_chunk_length = 33 and output_chunk_length = 6) the 2nd Error I get while backtesting is  

```
backtest = model.historical_forecasts(
    series=y_train_t,
    past_covariates=X_train_t,
    start=pd.Timestamp('2022/10/01') - pd.offsets.MonthBegin(12),   # I am setting the starting point 12 months before the last month 
                                                                                                             # in the dataset
    forecast_horizon=12,
    retrain=False,
    verbose=True
)

y_train_t.plot(label="actual")
backtest.plot(label="backtest (H=10)")
plt.legend()
```
I get an error saying `start` timestamp is too late in the series 0 to make any predictions with`overlap_end` set to `False``
**Expected behavior**

What I want : As mentioned earlier, I want to divide my dataset into training and validation of 24 train and 15 validation data to fit and train in the RNN. And then calculate and see the SMAPE in the validation set. I used historical forecasting thinking maybe thats the way to get SMAPE in DARTS for the validation and the training. I would prefer to get the SMAPE for the training as well. Please let me know if there is a better way. 

- Python version: 3.7.12
- Darts version 0.23.1

**Comment by dennisbader** [comment]
Hi @SireeshLimbu I strongly recommend to read our [guide on TorchForecastingModels](https://unit8co.github.io/darts/userguide/torch_forecasting_models.html#training). This explains everything about input/output chunks, validation series, and training/prediction.

**Comment by SireeshLimbu** [comment]
Hi @dennisbader . I have gone through the documentation. I have understood the last question I asked and have hence removed it. But I still don't get the Backtesting part.
